tests/test_converter.py

```diff
@@ -36,7 +36,6 @@
 import temporalio.api.common.v1
 import temporalio.common
 from temporalio.api.common.v1 import Payload, Payloads
-from temporalio.api.common.v1 import Payload as AnotherNameForPayload
```

Comment:
This looks like it was deliberate but also appears to have no purpose today.

Reply:
Agreed.